app-tasks/rf/src/rf/uploads/landsat_historical/factories.py

```diff
@@ -91,7 +91,7 @@ def create_scene(owner, prefix, landsat_id, config, datasource):
     (filename, cog_fname) = process_to_cog(prefix, gcs_prefix, landsat_id, config)
     s3_location = upload_file(owner, filename, cog_fname)
     logger.info('Creating image')
-    ingest_location = 's3://{}/{}'.format(data_bucket, urllib.quote(s3_location))
+    ingest_location = 's3://{}/{}'.format(data_bucket, quote(s3_location))
```

Comment:
I think you could be correct; however, I'm inclined to leave this since all that changed was the import. I also didn't test this since https://github.com/raster-foundry/raster-foundry/pull/5247 removes frontend support for CMR and we won't be testing historic landsat.